test/common.py

```diff
@@ -2349,6 +2349,8 @@ def setUpClass(cls):
     cls.harness_in_queue = multiprocessing.Queue()
     cls.harness_out_queue = multiprocessing.Queue()
     cls.harness_server = multiprocessing.Process(target=harness_server_func, args=(cls.harness_in_queue, cls.harness_out_queue, cls.PORT))
+    cls.SERVER_URL = f'http://localhost:{cls.PORT}'
+    cls.HARNESS_URL = f'{cls.SERVER_URL}/run_harness'
```

Comment:
Are these two lines needed?   Can we just use the class level declarations above?

Reply:
Yeah, can remove this. Will be needed for my later patch where the port is set differently.